orttraining/orttraining/test/training_api/core/checkpoint_test.cc

```diff
@@ -214,11 +214,11 @@ TEST(CheckpointApiTest, SaveOptimizerStateAsCheckpoint_ThenLoad_CUDA) {
 
   onnxruntime::training::test::training_api::SyntheticDataLoader data_loader;
   auto sample = onnxruntime::training::test::training_api::SyntheticSampleBatch();
-  sample.AddFloatInput(fc1_weight_shape);
-  sample.AddFloatInput(fc1_bias_shape);
-  sample.AddFloatInput(fc2_weight_shape);
-  sample.AddFloatInput(fc2_bias_shape);
-  data_loader.AddSyntheticSampleBatch(sample);
+  sample.AddFloatInput(gsl::make_span(fc1_weight_shape));
```

Comment:
is make_span necessary here? thought it would be implicitly convertible from std::vector

Reply:
removed.